webpack.config.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Webpack Config
+ * Based on angular2-webpack-starter by AngularClass
+ * https://github.com/angular-class/angular2-webpack-starter
+ */
+
+/*
+ * Helpers
+ */
+var sliceArgs = Function.prototype.call.bind(Array.prototype.slice);
+var toString  = Function.prototype.call.bind(Object.prototype.toString);
+var NODE_ENV  = process.env.NODE_ENV || 'development';
+var pkg = require('./package.json');
+
+// Polyfill
+Object.assign = require('object-assign');
+
+// Node
+var path = require('path');
+
+// NPM
+var webpack = require('webpack');
+
+// Webpack Plugins
+var OccurenceOrderPlugin = webpack.optimize.OccurenceOrderPlugin;
+var CommonsChunkPlugin   = webpack.optimize.CommonsChunkPlugin;
+var UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
+var DedupePlugin   = webpack.optimize.DedupePlugin;
+var DefinePlugin   = webpack.DefinePlugin;
+var BannerPlugin   = webpack.BannerPlugin;
+
+/*
+ * Config
+ */
+module.exports = {
+  devtool: 'source-map',
+  debug: true,
+  cache: true,
+
+  verbose: true,
+  displayErrorDetails: true,
+  context: __dirname,
+  stats: {
+    colors: true,
+    reasons: true
+  },
+
+  entry: {
+    'app': [
+      // Angular 2 Deps
+      '@reactivex/rxjs',
+      'zone.js',
+      'reflect-metadata',
+      // To ensure these modules are grouped together in one file
+      'angular2/angular2',
+      'angular2/core',
+      'angular2/router',
+      'angular2/http',
+      'json-formatter-js',
+      // Angular 2 ES6 browser shim
+      'es6-shim',
+      // Frontend
+      './frontend/batarangle'
+    ],
+    'devtools': ['./devtools/devtools'],
+    'backend': ['./backend/batarangle-backend'],
+    'entry': ['./backend/entry'],
+    'channel': ['./channel/channel']
+  },
+
+  // Config for our build files
+  output: {
+    path: root('build'),
+    filename: '[name].js',
+    sourceMapFilename: '[name].js.map',
+    chunkFilename: '[id].chunk.js'
+  },
+
+  resolve: {
+    root: __dirname,
+    extensions: ['','.ts','.js','.json'],
+    alias: {
+      'rx': '@reactivex/rxjs'
```

Comment:
This came with from the seed project. I believe the idea is that you can import from `rx` instead of `@reactivex/rxjs`. I just realized that we still have `rx` as a dependency so, it might not work. I'll remove it and test it out.